xml_inscriptions/transcribed/MI.AA.UM.KM.L.1452.xml

```diff
@@ -156,13 +157,14 @@
         
         
                 <div type="translation">
-                    <p>Aulus Marcius Antiochus, freedman of Aulus. In width, nine feet, in [depth,   feet.]</p>
+                    <p>Aulus Marcius Antiochus, freedman of Aulus. In width, nine feet, in [depth, X feet.]</p>
                 </div>
             
         
         
                 <div type="commentary">
                    <p> 2) A. LI. Anthiocus: <hi rend="italic">CIL</hi>  3) in [ag(ro) p(edes) novem)]: Schwartz</p>
```

Comment:
Do you want to change the formatting here so as to maintain consistency across the different commentary/app entries?